pkg/kubelet/cm/dra/claiminfo.go

```diff
@@ -25,40 +25,41 @@ import (
 	resourceapi "k8s.io/api/resource/v1alpha3"
 	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/apimachinery/pkg/util/sets"
-	"k8s.io/kubernetes/pkg/kubelet/cm/dra/state"
+	checkpoint "k8s.io/kubernetes/pkg/kubelet/cm/dra/checkpoint"
+	checkpointapi "k8s.io/kubernetes/pkg/kubelet/cm/dra/checkpoint/v1"
```

Comment:
I wouldn't expect us to have to pull both of these in here. Why is `ClaimInfoStateList` defined in `checkpoint` but `ClaimInfoState` is defined in `checkpoint/v1`?

Reply:
ClaimInfoStateList is here to avoid duplication in every API version. It's just a helper type for the list of ClaimInfoStates. If you think it shouldn't be outside of checkpoint/<version> I can either move it there or get rid of it and simply use []checkpoint.ClaimInfoState instead of it. Which way would you prefer?

Reply:
> I wouldn't expect us to have to pull both of these in here.

that's a result of getting rid of `state` subdirectory, I guess. Should I return it back and put state_checkpoint* files there?